### ISSUE TITLE
Data saved to database but not retrieve

### ISSUE THREAD

**Issue by HMarzban** [docs-plus/docs.plus#106]
I don't have this problem on the local machine!
Right now, the data is saved and retrieved from the indexeddb into the browser and also shared with other clients through this proxy.

**Comment by HMarzban** [comment]
Well, after lots of digging, I found the problem.
First, if we did not face any problem with saving data, it was because we used Indexdb, and all data was kept in the browser, and if two participants joined the document, the data was shared from the host with other clients from the browser to browsers. And if the host was not online, the other Guest did not have any content or the latest changes on the target page.

So in the first round, I thought the problem was related to the latest state change! But it was not.
Then I thought it was related to the backend infrastructure, so I changed the docker flow and tested many scenarios, but everything was okay locally. I did not have any issues saving data to the database.

After all, I understood that the problem does not link to the code base and infrastructure. I started digging in the server, and after changing the flow deployment flow, I finally found out the problem came from the environment variable!

I did everything I could to fix the stupid quote, but it wouldn't fix it, and it wouldn't change. In the end, I hard-coded the database type.

I spent a lot of time, and if I wanted to continue, I would have lost a lot of energy and time; the problem is with GitHub action env management. And if I delete all folders and install them again from scratch, this problem may be solved.
